@types/croppie @types to library inconsistency

HI @andy-ms, @sarunint, @dklmuc 

I tried using the `@types/croppie` I get the following issue at runtime

Uncaught TypeError: croppie_1.default is not a constructor.

**Import:** 
import  Croppie from "croppie";

**Usage:**
this.croppie = new Croppie(this.avatarImage,
                {
                    viewport: { width: 256, height: 256, type: this.viewPortType },
                    boundary: { width: 410, height: 410 },
                    showZoomer: true,
                });

In order to get around this problem I used the following.

**Import:** 
import * as Croppie from "croppie";

**Usage:**
this.croppie = new Croppie.Croppie(this.avatarImage,
                {
                    viewport: { width: 256, height: 256, type: this.viewPortType },
                    boundary: { width: 410, height: 410 },
                    showZoomer: true,
                });

This works fine at runtime but i get the following error during typescript compilation phase

error TS2339: Property 'Croppie' does not exist on type 'typeof ".../node_modules/@types/croppie/index"'.
[14:31:25] gulp-notify: [Error running Gulp] Error: ...\avatar-modal.ts(43,40): error TS2339: Property 'Croppie' does not exist on type 'typeof ".../node_modules/@types/croppie/index"'.

--- a/types/croppie/croppie-tests.ts
+++ b/types/croppie/croppie-tests.ts
@@ -1,4 +1,4 @@
-import Croppie from 'croppie';
+import * as Croppie from 'croppie';
 
 const c = new Croppie(document.getElementById('item'), {
     boundary: { width: 300, height: 300 },

--- a/types/croppie/index.d.ts
+++ b/types/croppie/index.d.ts
@@ -5,8 +5,12 @@
 //                 Sarun Intaralawan <https://github.com/sarunint>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export default class Croppie {
-    constructor(container: HTMLElement, options?: CroppieOptions);
+export as namespace Croppie;
+
+export = Croppie;
+
+declare class Croppie {
+    constructor(container: HTMLElement, options?: Croppie.CroppieOptions);
 
     bind(options: {
         url: string,
@@ -16,11 +20,11 @@ export default class Croppie {
         useCanvas?: boolean,
     }): Promise<void>;
 
-    result(options: ResultOptions & { type: 'base64' | 'canvas' }): Promise<string>;
-    result(options: ResultOptions & { type: 'html' }): Promise<HTMLElement>;
-    result(options: ResultOptions & { type: 'blob' }): Promise<Blob>;
-    result(options: ResultOptions & { type: 'rawcanvas' }): Promise<HTMLCanvasElement>;
-    result(options?: ResultOptions): Promise<HTMLCanvasElement>;
+    result(options: Croppie.ResultOptions & { type: 'base64' | 'canvas' }): Promise<string>;
+    result(options: Croppie.ResultOptions & { type: 'html' }): Promise<HTMLElement>;
+    result(options: Croppie.ResultOptions & { type: 'blob' }): Promise<Blob>;
+    result(options: Croppie.ResultOptions & { type: 'rawcanvas' }): Promise<HTMLCanvasElement>;
+    result(options?: Croppie.ResultOptions): Promise<HTMLCanvasElement>;
 
     rotate(degrees: 90 | 180 | 270 | -90 | -180 | -270): void;
 
@@ -29,28 +33,30 @@ export default class Croppie {
     destroy(): void;
 }
 
-export type CropType = 'square' | 'circle';
+declare namespace Croppie {
+    type CropType = 'square' | 'circle';
 
-export type Format = 'jpeg' | 'png' | 'webp';
+    type Format = 'jpeg' | 'png' | 'webp';
 
-export type Type = 'canvas' | 'base64' | 'html' | 'blob' | 'rawcanvas';
+    type Type = 'canvas' | 'base64' | 'html' | 'blob' | 'rawcanvas';
 
-export interface ResultOptions {
-    type?: Type;
-    size?: 'viewport' | 'original' | { width: number, height: number };
-    format?: Format;
-    quality?: number;
-    circle?: boolean;
-}
+    interface ResultOptions {
+        type?: Type;
+        size?: 'viewport' | 'original' | { width: number, height: number };
+        format?: Format;
+        quality?: number;
+        circle?: boolean;
+    }
 
-export interface CroppieOptions {
-    boundary?: { width: number, height: number };
-    customClass?: string;
-    enableExif?: boolean;
-    enableOrientation?: boolean;
-    enableZoom?: boolean;
-    enforceBoundary?: boolean;
-    mouseWheelZoom?: boolean;
-    showZoomer?: boolean;
-    viewport?: { width: number, height: number, type?: CropType };
+    interface CroppieOptions {
+        boundary?: { width: number, height: number };
+        customClass?: string;
+        enableExif?: boolean;
+        enableOrientation?: boolean;
+        enableZoom?: boolean;
+        enforceBoundary?: boolean;
+        mouseWheelZoom?: boolean;
+        showZoomer?: boolean;
+        viewport?: { width: number, height: number, type?: CropType };
+    }
 }

